web/js/animation/model.js

```diff
@@ -99,12 +99,7 @@ wv.anim.model = wv.anim.model || function (models, config) {
    *
    */
   self.save = function (state) {
-    var activeState;
-    var playStates;
-    var options;
-    var newState;
-
-    rangeState = self.rangeState;
+    var rangeState = self.rangeState;
     // state.a = state.a || [];
```

Comment:
I think that if a comment is something you would like to get rid of, do it. However, it is a little outside the scope of the refactor and shouldn't be a requirement for getting a PR accepted. The reason I am removing unused vars is because they are picked up by the linter and resolving linting errors (when non-invasive) has been very helpful in this refactor.

Reply:
Ok, it's just an idea that came up, since you're removing code that's referenced in the comments, it seems like it might be a good time to remove the comments too. It doesn't need to be addressed in this PR. Your PR is good without it, I just thought it would be an easy relevant thing to do at the same time, that's why I mentioned it.